tests/framework/framework.go

```diff
@@ -78,6 +83,10 @@ type Framework struct {
 
 // initialize run-time flags
 func init() {
+	// By accessing something in the ginkgo_reporters package, we are ensuring that the init() is called
```

Comment:
Is this specific to red hat qe or generally useful?

Reply:
Its specific to RH QE I think, well it generates a junit.xml file so I guess its more general, but its used for QE.